### DISCUSSION TITLE
Idea/Suggestion Use Windows 11 Icon Tray, Search, and Task View Icon

### DISCUSSION THREAD

**Discussion by sapphired18** [valinet/ExplorerPatcher#260] [Ideas]
![image](https://user-images.githubusercontent.com/58063311/139968032-689274c5-8a89-44a7-b4e3-6e4542d50be1.png)
Change Cortana to Widgets Icon too!
![image](https://user-images.githubusercontent.com/58063311/139968039-418ba887-4128-48f6-ae34-2d1bf9b73881.png)

**Comment by valinet** [comment]
It’s on my to-do list at some point! It’s always open for contributions, as well!!!

Thanks!
